.github/workflows/Documentation.yml

```
@@ -12,12 +12,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@latest
```
@jishnub I know next to nothing about github actions, but the need to change versions manually like this seems to be hard to maintain in the long run.  Would using `latest` ensure it will keep working in the long term?  If not, then just cancel this suggestion.  Same question with `cache` below.
I am extrapolating from `setup-julia@latest` without knowing if `latest` is something generally supported...

---

I tried this in #62 and it failed the Documenter build, so never mind...